torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -220,6 +221,7 @@ def __init__(
             # TODO: example_inputs causes more cache misses than necessary
             # with dynamic shapes, because this is before we add
             # symints to tensor metadata. Improve this later.
+            FxGraphCache._check_can_cache(gm)
```

Comment:
was this a bugfix?

Reply:
There are certain inputs to the cache that cause issues when trying to pickle when building a cache key. Specifically, if the inductor config has a post_grad_custom_pre_pass or post_grad_custom_post_pass, we try to cache that as part of the AOTAutogradCache key. 

But if the  post_grad_custom_post_pass isn't a specific type, it's unserializable, leading to a pickle error. 

Technically, this is caught by the bigger try catch outside of the key generation, but rather than catching it there, it's better to raise the more accurate exception, which is that FxGraphCache only supports specific configs. 

https://github.com/pytorch/pytorch/blob/e0d97e936a2baed40224ee3c82d68ec9897c4693/torch/_inductor/codecache.py#L1398-L1401

Reply:
ah yeah, the custom inductor pass config :(